--- "/opt/awsvpnclient/AWS VPN Client.deps.json"
+++ "/opt/awsvpnclient/AWS VPN Client.deps.json"
@@ -695,9 +695,6 @@
           }
         },
         "native": {
-          "createdump": {
-            "fileVersion": "0.0.0.0"
-          },
           "libSystem.Globalization.Native.so": {
             "fileVersion": "0.0.0.0"
           },
@@ -722,20 +719,11 @@
           "libcoreclr.so": {
             "fileVersion": "0.0.0.0"
           },
-          "libcoreclrtraceptprovider.so": {
-            "fileVersion": "0.0.0.0"
-          },
           "libhostfxr.so": {
             "fileVersion": "0.0.0.0"
           },
           "libhostpolicy.so": {
             "fileVersion": "0.0.0.0"
-          },
-          "libmscordaccore.so": {
-            "fileVersion": "0.0.0.0"
-          },
-          "libmscordbi.so": {
-            "fileVersion": "0.0.0.0"
           }
         }
       },
@@ -3241,4 +3229,4 @@
       "base"
     ]
   }
-}
\ No newline at end of file
+}
