--- "/opt/awsvpnclient/AWS VPN Client.deps.json"	2023-03-07 16:10:12.674752986 +0100
+++ "/opt/awsvpnclient/AWS VPN Client.deps.json"	2023-03-07 16:29:38.741012201 +0100
@@ -11,8 +11,6 @@
         "dependencies": {
           "AWSVPNClient.Core": "2.0.0",
           "GtkSharp": "3.22.25.128",
-          "System.Data.SQLite": "1.0.115",
-          "System.Data.SQLite.Core": "1.0.115",
           "AWSVPNClient.Core.Reference": "2.0.0.0",
           "runtimepack.Microsoft.NETCore.App.Runtime.linux-x64": "8.0.10"
         },
@@ -696,9 +694,6 @@
           }
         },
         "native": {
-          "createdump": {
-            "fileVersion": "0.0.0.0"
-          },
           "libSystem.Globalization.Native.so": {
             "fileVersion": "0.0.0.0"
           },
@@ -723,20 +718,11 @@
           "libcoreclr.so": {
             "fileVersion": "0.0.0.0"
           },
-          "libcoreclrtraceptprovider.so": {
-            "fileVersion": "0.0.0.0"
-          },
           "libhostfxr.so": {
             "fileVersion": "0.0.0.0"
           },
           "libhostpolicy.so": {
             "fileVersion": "0.0.0.0"
-          },
-          "libmscordaccore.so": {
-            "fileVersion": "0.0.0.0"
-          },
-          "libmscordbi.so": {
-            "fileVersion": "0.0.0.0"
           }
         }
       },
@@ -1058,28 +1044,6 @@
           }
         }
       },
-      "System.Data.SQLite/1.0.115": {
-        "dependencies": {
-          "System.Data.SQLite.Core": "1.0.115",
-          "System.Data.SQLite.EF6": "1.0.115"
-        }
-      },
-      "System.Data.SQLite.Core/1.0.115": {
-        "dependencies": {
-          "Stub.System.Data.SQLite.Core.NetStandard": "1.0.115"
-        }
-      },
-      "System.Data.SQLite.EF6/1.0.115": {
-        "dependencies": {
-          "EntityFramework": "6.4.4"
-        },
-        "runtime": {
-          "lib/netstandard2.1/System.Data.SQLite.EF6.dll": {
-            "assemblyVersion": "1.0.115.0",
-            "fileVersion": "1.0.115.0"
-          }
-        }
-      },
       "System.Diagnostics.DiagnosticSource/4.7.1": {},
       "System.Drawing.Common/4.7.0": {
         "dependencies": {
@@ -1308,8 +1272,6 @@
           "Serilog.Sinks.File": "5.0.0",
           "Serilog.Sinks.File.Header": "1.0.2",
           "SymbolicLinkSupport": "1.2.0",
-          "System.Data.SQLite": "1.0.115",
-          "System.Data.SQLite.Core": "1.0.115",
           "System.IO.FileSystem.AccessControl": "5.0.0",
           "System.Runtime.InteropServices": "4.3.0",
           "System.ServiceModel.Http": "4.8.1",
@@ -1672,13 +1634,6 @@
       "path": "serilog.sinks.file.header/1.0.2",
       "hashPath": "serilog.sinks.file.header.1.0.2.nupkg.sha512"
     },
-    "Stub.System.Data.SQLite.Core.NetStandard/1.0.115": {
-      "type": "package",
-      "serviceable": true,
-      "sha512": "sha512-afpVRUKjuLZ17W2iv779YORpWQN40rJ82g/ASC6Yq1J4emO+8hL316AISf5nJWfXecygLdqevzXxQZR7z2yRww==",
-      "path": "stub.system.data.sqlite.core.netstandard/1.0.115",
-      "hashPath": "stub.system.data.sqlite.core.netstandard.1.0.115.nupkg.sha512"
-    },
     "SymbolicLinkSupport/1.2.0": {
       "type": "package",
       "serviceable": true,
@@ -1714,27 +1669,6 @@
       "path": "system.data.sqlclient/4.8.1",
       "hashPath": "system.data.sqlclient.4.8.1.nupkg.sha512"
     },
-    "System.Data.SQLite/1.0.115": {
-      "type": "package",
-      "serviceable": true,
-      "sha512": "sha512-aUVnApBCZ+8VNzTquKAd0CbDl2fMofeMQuRPBdYDHgmEn7n38RZ600MpsbeAEPXvLNocWI/T88mALMDAqeTX1w==",
-      "path": "system.data.sqlite/1.0.115",
-      "hashPath": "system.data.sqlite.1.0.115.nupkg.sha512"
-    },
-    "System.Data.SQLite.Core/1.0.115": {
-      "type": "package",
-      "serviceable": true,
-      "sha512": "sha512-OuzSKnAnXtbsB7MUddjLiLAUEuEu8kJco3jWllZs+zgYmuiK5VHUHafD74C65HwnnmWVHUvFBrJ0NDAKINuGNQ==",
-      "path": "system.data.sqlite.core/1.0.115",
-      "hashPath": "system.data.sqlite.core.1.0.115.nupkg.sha512"
-    },
-    "System.Data.SQLite.EF6/1.0.115": {
-      "type": "package",
-      "serviceable": true,
-      "sha512": "sha512-dzPRxNdxj5/k7hd3Qd3aZxYROHy+cqhiv4NJjsdhD8lnJ7zcoXiy36AyYe85nsohxuodYjuYHj0pMzGaC28B7A==",
-      "path": "system.data.sqlite.ef6/1.0.115",
-      "hashPath": "system.data.sqlite.ef6.1.0.115.nupkg.sha512"
-    },
     "System.Diagnostics.DiagnosticSource/4.7.1": {
       "type": "package",
       "serviceable": true,
