--- /usr/share/applications/awsvpnclient.desktop	2022-05-17 23:51:35.000000000 +0200
+++ /usr/share/applications/awsvpnclient.desktop	2022-05-17 23:51:35.000000000 +0200
@@ -3,10 +3,10 @@
 Encoding=UTF-8
 Name=AWS VPN Client
 Comment=AWS VPN Client
-Exec=/opt/awsvpnclient/AWS\ VPN\ Client %u
+Exec=/opt/awsvpnclient/AWSVPNClient
 Path=/opt/awsvpnclient
-Icon=acvc-64.png
+Icon=acvc-64
 Terminal=false
-Categories=Network;VPN;
+Categories=Network;X-VPN;
 Keywords=vpn;aws;
 
